app/assets/stylesheets/components/multi-input.scss

```diff
@@ -0,0 +1,21 @@
+.c-input--multi {
+  &__wrapper-border:focus-within {
+    @extend %form-styling-focus;
+
+    .c-input--multi__input {
```

Comment:
`c-input--multi__input` this classname may look a bit weird but was struggling to come up with an alternative. Open to thoughts and suggestions.